qupulse/_program/waveforms.py

```diff
@@ -243,6 +243,43 @@ def __repr__(self):
         return f'{type(self).__name__}(channel={self._channel_id}, waveform_table={self._table})'
 
 
+class ConstantWaveform(Waveform):  
+    def __init__(self, duration: float, amplitude: Any, channel: ChannelID):
+        """ Create a qupulse waveform from a qupulse template that has a QI waveform inside """
+        self._duration = duration
+        self._amplitude = amplitude
+        self._channel = channel
+
+        self._is_constant_waveform = True
```

Comment:
What is the purpose of this field?

Reply:
It is used by our backend (for the HDAWG8) to get a qupulse template represents a constant pulse/waveform.

Reply:
I am thinking about adding some inspection properties such as `is_constant` and `is_zero` in the mid-term. Until then this is a good solution, I think.